modules/apps/27-interchain-accounts/controller/client/cli/query.go

```diff
@@ -11,6 +11,40 @@ import (
 	"github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/controller/types"
 )
 
+// GetCmdQueryInterchainAccount returns the command handler for the controller submodule parameter querying.
+func GetCmdQueryInterchainAccount() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "interchain-account [connection-id] [owner]",
```

Comment:
Should we maybe swap the order?
```suggestion
		Use:     "interchain-account [owner] [connection-id]",
```